server/controllers/alertsController.ts

```diff
@@ -0,0 +1,44 @@
+import { Request, Response } from 'express'
+import { PagedListQueryParams } from '../interfaces/prisonApi/pagedList'
+import AlertsPageService from '../services/alertsPageService'
+import { mapHeaderData } from '../mappers/headerMappers'
+import { PrisonerSearchService } from '../services'
+
+/**
+ * Parse request for alerts page and orchestrate response
+ */
+export default class AlertsController {
+  private prisonerSearchService: PrisonerSearchService
+
+  private alertsPageService: AlertsPageService
+
+  constructor(clientToken: string) {
+    this.prisonerSearchService = new PrisonerSearchService(clientToken)
+    this.alertsPageService = new AlertsPageService(clientToken)
+  }
```

Comment:
IMO if we're instantiating controllers it'd be good to do dependency inversion at the same time and pass in the services it's dependent on. Otherwise tests that want to control the responses of these can get a bit more messy and I've found jests module mocking a bit hit and miss. 

This also means we need to make interfaces for the services so that the class is dependent on those rather than the actual implementations, otherwise we can't pass them in.